crypto/core_fetch.c

```diff
@@ -30,7 +30,31 @@ static void ossl_method_construct_this(OSSL_PROVIDER *provider,
 {
     struct construct_data_st *data = cbdata;
     void *method = NULL;
+    int global_store = data->force_store || !no_store;
 
+    /*
+     * Check to see if a method has already been constructed from this
+     * implementation.  It's pointless to do this check if we're not
+     * supposed to store the method in the global store.
+     *
+     * Note that while data->mcm->put() takes a property *definition*,
+     * while data->mcm->get() takes a property *query*, but we're using
+     * the property *definition* string as a property query here.
+     * This is possible because property definitions have a syntax that
+     * can also be used for a property query with equality tests.  We do
+     * this to look up already existing methods matching the implementation
+     * that we just for from the provider (via |algo|),
```

Comment:
much better this way, lust the last few words need to be unscrambled
`that we just for from the provider` and should this sentence end with a period ?

Reply:
Typo.  Fixed.